packages/expo-cli/src/commands/client/index.ts

```diff
@@ -202,13 +202,13 @@ export default function (program: Command) {
         }
 
         let email;
-        if (user) {
+        if (user && user.kind === 'user') {
```

Comment:
I did this because typescript doesn't like the alternatives 😅   With the robot user we have 2 types of users, both having the `.kind` property. It also infers the type when using this property. If you use a property that's only defined in a single type, without guarding, it will complain that it might not exist.

![Screenshot 2020-08-26 at 15 58 57](https://user-images.githubusercontent.com/1203991/91313179-5b97e700-e7b5-11ea-9f22-b351449019f0.png)

Even if you use `if (user && (user as User).email)` or just `if ((user as User)?.email)` it complains later on:

![Screenshot 2020-08-26 at 15 59 15](https://user-images.githubusercontent.com/1203991/91313239-723e3e00-e7b5-11ea-844e-3d84783d01f4.png)

Which is why I used the `.kind` here 😄